### PR TITLE
add get_version endpoint

### PR DESCRIPTION
**Pull request by fcollman**

### CODE REVIEW

**Comment by bdpedigo** [comment]
the remaining test failure (I believe) is something I fixed in https://github.com/CAVEconnectome/CAVEclient/pull/250

**Comment by bdpedigo** [comment]
i also moved the version endpoint to common but can move back if there was some reason for it being elsewhere

File: tests/conftest.py [hunk 1]
```diff
@@ -46,9 +46,12 @@ def myclient():
     # narrowly scoped to this function
     version_url = f"{TEST_LOCAL_SERVER}/segmentation/api/version"
     responses.add(responses.GET, version_url, json="2.15.0", status=200)
-
     client.chunkedgraph  # this will trigger the version check
 
+    mat_version_url = f"{TEST_LOCAL_SERVER}/materialize/version"
+    responses.add(responses.GET, mat_version_url, json="4.30.1", status=200)
+    client.materialize
```
Comment (fcollman) [inline tests/conftest.py:53]: this is i think a mistake

Comment (bdpedigo) [reply tests/conftest.py:53]: can you elaborate? i put this in bc now the client sees there's an endpoint for version and tries to check it

**Pull request merged**